modules/auth/sso/oauth2.go

```diff
@@ -12,6 +12,7 @@ import (
 
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/middlewares"
```

Comment:
`middlewares` should be `middleware` as we use singular for others

Reply:
#14480